specs/StartInBackgroundMode.md

```diff
@@ -0,0 +1,101 @@
+CreationPriority
+===
+
+# Background
+WebView2 has functionality enabled by default which raises the launch priority of WebView2
+processes to ensure that WebView2 processes are given higher CPU priority and access to resources
+during creation, and that can bring improvements on time spent creating a WebView2 instance. Since 
+this improvement is at the cost of taking higher CPU priority, we introduce the `CreationPriority` 
+property to give developers an option to decide by themselves.
+
+# Description
+* Setting `CreationPriority` to `COREWEBVIEW2_CREATION_PRIORITY_NORMAL` will signal WebView2 
+to have normal priority compared to other processes during the creation of the WebView2 browser 
+process.
+* Setting `CreationPriority` to `COREWEBVIEW2_CREATION_PRIORITY_HIGH` will attempt to enable 
+raise the priority of the creation of the WebView2 browser process.
+* Default value is `COREWEBVIEW2_CREATION_PRIORITY_HIGH`.
+# Examples
+## Win32 C++
+```cpp
+auto options = Microsoft::WRL::Make<CoreWebView2EnvironmentOptions>();
+/// Set the WebView2 browser process to have normal priority during 
+/// creation, which avoids to impact other host application 
+/// processes acquiring system resources.
+options->put_CreationPriority(COREWEBVIEW2_CREATION_PRIORITY_NORMAL);
+HRESULT hr = CreateCoreWebView2EnvironmentWithOptions(
+  nullptr, m_userDataFolder.c_str(), options.Get(),
+  Callback<ICoreWebView2CreateCoreWebView2EnvironmentCompletedHandler>(
+     this, &AppWindow::OnCreateEnvironmentCompleted).Get());
+```
+## WinRT and .NET
+```c#
+CoreWebView2EnvironmentOptions options = new CoreWebView2EnvironmentOptions();
+options.CreationPriority = COREWEBVIEW2_CREATION_PRIORITY_NORMAL;
+CoreWebView2Environment Environment = 
+    await CoreWebView2Environment.CreateAsync(
+        BrowserExecutableFolder, UserDataFolder, options)
+```
+
+
+# API Details
+## Win32 C++
+```cpp
+/// Specifies the type of webview2 creation priority for the
+/// `ICoreWebView2EnvironmentOptions3:CreationPriority` option.
+[v1_enum]
+typedef enum COREWEBVIEW2_CREATION_PRIORITY {
+  COREWEBVIEW2_CREATION_PRIORITY_NORMAL,
+  COREWEBVIEW2_CREATION_PRIORITY_HIGH,
+} COREWEBVIEW2_CREATION_PRIORITY;
+
+[uuid(efb58776-32bd-11ed-a261-0242ac120002), object, pointer_default(unique)]
+interface ICoreWebView2EnvironmentOptions3 : ICoreWebView2EnvironmentOptions2 {

```

Comment:
Talking to @LiangTheDev we should move CreationPriority to the CoreWebView2Environment. In the future we may want to support changing the priority during creation.